var/spack/repos/builtin/packages/root/package.py

```diff
@@ -31,15 +31,24 @@ class Root(CMakePackage):
     """ROOT is a data analysis framework."""
 
     homepage = "https://root.cern.ch"
-    url      = "https://root.cern.ch/download/root_v6.09.02.source.tar.gz"
+    url      = "https://root.cern/download/root_v6.12.06.source.tar.gz"
+    list_url = "https://root.cern/download/"
+
+    # ###################### Versions ##########################
+
+    # Master branch
+    version('master', git="https://github.com/root-project/root.git",
+        branch='master')
 
     # Development versions
-    version('6.09.02', '4188dfeafb72df339a3d688fe92f57ec')
```

Comment:
We should probably keep this version in case someone already installed it and wants to be able to reproduce it. Can we move it to "Old versions"?